stackslib/src/chainstate/nakamoto/tests/node.rs

```diff
@@ -95,17 +95,21 @@ impl TestStacker {
 
     /// make a set of stackers who will share a single signing key and stack with
     /// `Self::DEFAULT_STACKER_AMOUNT`
-    pub fn common_signing_set(test_signers: &TestSigners) -> Vec<TestStacker> {
-        let mut signing_key_seed = test_signers.num_keys.to_be_bytes().to_vec();
+    pub fn common_signing_set() -> (TestSigners, Vec<TestStacker>) {
```

Comment:
Might want to update the comment about about "sharing a single signing key"

Reply:
They do still share a single signer key. This function does the same thing as before, I just refactored it because previously `TestSigners` didn't actually store signer keys - it was just using `TestSigner.num_keys` (which was a constant value of 10) to determine the number of stackers.